MAINTAINERS.md

```diff
@@ -90,7 +88,8 @@ An automated list can be found [here](https://github.com/orgs/tldr-pages/people)
   [17 December 2019](https://github.com/tldr-pages/tldr/issues/3663) — [18 June 2020](https://github.com/tldr-pages/tldr/issues/4113)
 - Lucas Schneider ([@schneiderl](https://github.com/schneiderl)):
   [17 January 2020](https://github.com/tldr-pages/tldr/issues/3764) — [3 February 2021](https://github.com/tldr-pages/tldr/issues/5224)
-
+- **Axel Navarro ([@navarroaxel](https://github.com/navarroaxel))**:
+  [5 October 2020](https://github.com/tldr-pages/tldr/issues/4504) — [7 April 2021](https://github.com/tldr-pages/tldr/issues/5703)
```

Comment:
```suggestion
```

Reply:
:thinking: Why the deletion?

Reply:
See, look at the [diff](https://github.com/tldr-pages/tldr/pull/5730/files?short_path=39da3bd#diff-39da3bd6270d44ea37b6ed50bd42eeb9d93ac5e1639645871a69cbe08cbe29de).  You added it twice

Reply:
Yeah, I'm not understanding this either @bl-ue. Generally when someone moves from 1 role to another we put an end date on their time in that role, and then create a new entry under the new role.

Reply:
Oh, I see why you're not understanding; it looked to me like you added a wrong entry, but it's really because you forgot to unbold it:
```suggestion
- Axel Navarro ([@navarroaxel](https://github.com/navarroaxel)):
  [5 October 2020](https://github.com/tldr-pages/tldr/issues/4504) — [7 April 2021](https://github.com/tldr-pages/tldr/issues/5703)
```